core/rest/rest_api.json

```diff
@@ -5,7 +5,7 @@
         "description": "Interact with the enterprise blockchain through Hyperledger Fabric API",
         "version": "1.0.0"
     },
-    "host": "127.0.0.1:3000",
+    "host": "127.0.0.1:5000",
```

Comment:
@srderson you're right. If we're remapping the host port to 5000, the Swagger UI would now be accessed on port 5000 on the host. So this change makes sense.